src/middleware/packages/ldp/mixins/orphan-files-deletion.js

```diff
@@ -0,0 +1,62 @@
+const { CronJob } = require('cron');
+const urlJoin = require('url-join');
+
+module.exports = {
+  settings: {
+    orphanFilesDeletion: {
+      baseUrl: undefined,
+      cronJob: {
+        time: '0 0 4 * * *', // Every night at 4am
+        timeZone: 'Europe/Paris'
+      }
+    }
+  },
+  dependencies: ['triplestore', 'ldp.resource'],
+  actions: {
+    async checkOrphanFiles(ctx) {
+      try {
+        this.logger.info('OrphanFilesDeletion - Check...');
+
+        const containerUri = urlJoin(this.settings.orphanFilesDeletion.baseUrl, this.settings.path);
```

Comment:
If you use the `getContainerUri` method of the ControlledContainerMixin, you don't need the `baseUrl` setting:
https://github.com/assemblee-virtuelle/semapps/blob/8687009431024704e24db838a2c9ba69f578374c/src/middleware/packages/ldp/mixins/controlled-container.js#L94-L96